specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/2022-07-31/iotcentral.json

```diff
@@ -0,0 +1,6113 @@
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/enrollmentGroups/{enrollmentGroupId}/certificates/{entry}/generateVerificationCode": {
+      "post": {
+        "operationId": "EnrollmentGroups_GenerateVerificationCodeX509",
+        "summary": "Generate a verification code for the primary or secondary x509 certificate of an enrollment group",
+        "description": "Generate a verification code for the primary or secondary x509 certificate of an enrollment group.",
+        "x-ms-examples": {
+          "Generate primary or secondary X509 verification code": {
+            "$ref": "./examples/enrollmentgroups_generateverificationcodex509.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/EnrollmentGroupId"
+          },
+          {
+            "$ref": "#/parameters/Entry"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/X509VerificationCode"
+            }
+          },
+          "default": {
+            "description": "An error response received from the IoT Central Service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/enrollmentGroups/{enrollmentGroupId}/certificates/{entry}/verify": {
+      "post": {
+        "operationId": "EnrollmentGroups_VerifyX509",
+        "summary": "Verify the primary or secondary x509 certificate of an enrollment group",
+        "description": "Verify the primary or secondary x509 certificate of an enrollment group by providing a certificate with the signed verification code.",
+        "x-ms-examples": {
+          "Verify primary or secondary X509 verification code": {
+            "$ref": "./examples/enrollmentgroups_verifyx509.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/EnrollmentGroupId"
+          },
+          {
+            "$ref": "#/parameters/Entry"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Certificate for the signed verification code.",
+            "schema": {
+              "$ref": "#/definitions/X509VerificationCertificate"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
```

Comment:
A 200 response should have a response body schema. If there is no body in the response then please use `204 NoContent`.